src/firefox/index.js

```diff
@@ -44,26 +47,26 @@ export function run(
       return new Promise((resolve) => {
         let firefox = results.process;
 
-        console.log(`Executing Firefox binary: ${results.binary}`);
-        console.log(`Executing Firefox with args: ${results.args.join(' ')}`);
+        log.debug(`Executing Firefox binary: ${results.binary}`);
+        log.debug(`Executing Firefox with args: ${results.args.join(' ')}`);
 
         firefox.on('error', (error) => {
           // TODO: show a nice error when it can't find Firefox.
           // if (/No such file/.test(err) || err.code === 'ENOENT') {
-          console.log(`Firefox error: ${error}`);
+          log.error(`Firefox error: ${error}`);
           throw error;
         });
 
         firefox.stderr.on('data', (data) => {
-          console.error(`stderr: ${data.toString().trim()}`);
+          log.error(`stderr: ${data.toString().trim()}`);
         });
 
         firefox.stdout.on('data', (data) => {
-          console.log(`stdout: ${data.toString().trim()}`);
+          log.info(`stdout: ${data.toString().trim()}`);
```

Comment:
Even when the logger is not in verbose mode, we probably want to make it clear when the logs come from the Firefox binary, how about add it here as part of the log message? e.g.

```
        firefox.stderr.on('data', (data) => {
          log.error(`stderr: ${data.toString().trim()}`);
          log.error(`${results.binary} stderr: ${data.toString().trim()}`);
        });

        firefox.stdout.on('data', (data) => {
          log.info(`stdout: ${data.toString().trim()}`);
          log.debug(`${results.binary} stdout: ${data.toString().trim()}`);
        });
```